.github/workflows/build_docs

```diff
@@ -0,0 +1,38 @@
+name: build_docs
```

Comment:
This change seems unnecessary if the intended change is only to change some docstrings in AGSWellbores.py. If it is intended, let me know and let's discuss because it's failing on the PR and also redundant conflicting with the docs build I already added